wit/ephemeral/outbound-redis.wit

```diff
@@ -0,0 +1,12 @@
+use * from redis-types
+
+// Publish a Redis message to the specificed channel and return an error, if any.
+publish: function(address: string, channel: string, payload: payload) -> expected<_, error>
+
+// Get the value of a key. An error is returned if the value stored at key is not
```

Comment:
The comment here (re: an error if the value is not a string) is no longer accurate.